bemani/backend/popn/peace.py

```diff
@@ -27,9 +27,133 @@ class PopnMusicPeace(PopnMusicModernBase):
     def previous_version(self) -> PopnMusicBase:
         return PopnMusicUsaNeko(self.data, self.config, self.model)
 
+    @classmethod
+    def get_settings(cls) -> Dict[str, Any]:
+        """
+        Return all of our front-end modifiably settings.
+        """
+        return {
+            'ints': [
+                {
+                    'name': 'Music Open Phase',
+                    'tip': 'Default music phase for all players.',
+                    'category': 'game_config',
+                    'setting': 'music_phase',
+                    'values': {
+                        # The value goes to 23 now, but it starts where usaneko left off at 11
+                        # Unlocks a total of 53 songs
+                        12: 'No music unlocks',
+                        13: 'Phase 1',
+                        14: 'Phase 2',
+                        15: 'Phase 3',
+                        16: 'Phase 4',
+                        17: 'Phase 5',
+                        18: 'Phase 6',
+                        19: 'Phase 7',
+                        20: 'Phase 8',
+                        21: 'Phase 9',
+                        22: 'Phase 10',
+                        23: 'Phase MAX',
+                    }
+                },
+                {
+                    'name': 'NAVI-Kun Event Phase',
+                    'tip': 'NAVI-Kun event phase for all players.',
+                    'category': 'game_config',
+                    'setting': 'navikun_phase',
```

Comment:
This doesn't appear used.

Reply:
So much for double checking and testing!